src/browser/pages/vscode.ts

```diff
@@ -72,20 +72,22 @@ export function getNlsConfiguration(_document: Document, base: string) {
 
     type LoadBundleCallback = (_: undefined, result?: string) => void
 
-    nlsConfig.loadBundle = (bundle: string, _language: string, cb: LoadBundleCallback): void => {
+    nlsConfig.loadBundle = async (bundle: string, _language: string, cb: LoadBundleCallback): Promise<void> => {
```

Comment:
🧹 Originally, this function was not using async/await. I then realized we had the `fetch` call and the `.then` chain. I refactored this so it was easy to test. Now it's clear to the caller this function has asynchronous logic.

Reply:
nice :+1: